controller/src/main/java/io/pravega/controller/server/rpc/auth/PravegaInterceptor.java

```diff
@@ -50,11 +51,12 @@
 
         Map<String, String> paramMap = new HashMap<>();
         headers.keys().stream()
-               .filter(key -> !key.endsWith(Metadata.BINARY_HEADER_SUFFIX))
+               .filter(key -> key.startsWith(Credentials.AUTH_HANDLER_PREFIX))
                .forEach(key -> {
                    try {
-                       paramMap.put(key,
+                       paramMap.put(key.substring(Credentials.AUTH_HANDLER_PREFIX.length()),
                                headers.get(Metadata.Key.of(key, Metadata.ASCII_STRING_MARSHALLER)));
+                       log.warn(key + " " + headers.get(Metadata.Key.of(key, Metadata.ASCII_STRING_MARSHALLER)));
```

Comment:
This smells like log spam. Why is it needed?

Reply:
Fixed.